flow/raster_cache.cc

```diff
@@ -231,13 +231,25 @@ RasterCacheResult RasterCache::Get(const SkPicture& picture,
                                    const SkMatrix& ctm) const {
   PictureRasterCacheKey cache_key(picture.uniqueID(), ctm);
   auto it = picture_cache_.find(cache_key);
-  return it == picture_cache_.end() ? RasterCacheResult() : it->second.image;
+  if (it == picture_cache_.end()) {
+    return RasterCacheResult();
+  }
+  Entry& entry = const_cast<Entry&>(it->second);
```

Comment:
Can we dry this up just a little bit by creating a method that takes a key and returns a cache result whose entry has its access counted bumped already?